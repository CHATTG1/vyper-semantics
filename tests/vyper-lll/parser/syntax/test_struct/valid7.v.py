mom: {a: {c: num}[3], b: num}
@public
def foo():
    nom: {c: num}[3]
    self.mom = {a: nom, b: 5}
