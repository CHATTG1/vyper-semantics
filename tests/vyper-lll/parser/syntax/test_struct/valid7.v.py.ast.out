$%seq ( $return ( 0 , $lll ( $%seq ( $%seq ( $mstore ( 28 , $calldataload ( 0 ) ) , $mstore ( 32 , 1461501637330902918203684832716283019655932542976 ) , $mstore ( 64 , 170141183460469231731687303715884105727 ) , $mstore ( 96 , -170141183460469231731687303715884105728 ) , $mstore ( 128 , 1701411834604692317316873037158841057270000000000 ) , $mstore ( 160 , -1701411834604692317316873037158841057280000000000 ) ) , $if ( $eq ( $mload ( 0 ) , 3264763256 ) , $%seq ( $pass , $assert ( $iszero ( $callvalue ) ) , $pass , $with ( $_L , $sha3_32 ( 0 ) , $%seq ( $with ( $_L , $sha3_32 ( $add ( $_L , 0 ) ) , $with ( $_R , 320 , $%seq ( $with ( $_L , $sha3_32 ( $add ( $_L , $uclamplt ( 0 , 3 ) ) ) , $with ( $_R , $add ( $mul ( 32 , $uclamplt ( 0 , 3 ) ) , $_R ) , $%seq ( $sstore ( $add ( $_L , 0 ) , $mload ( $add ( 0 , $_R ) ) ) ) ) ) , $with ( $_L , $sha3_32 ( $add ( $_L , $uclamplt ( 1 , 3 ) ) ) , $with ( $_R , $add ( $mul ( 32 , $uclamplt ( 1 , 3 ) ) , $_R ) , $%seq ( $sstore ( $add ( $_L , 0 ) , $mload ( $add ( 0 , $_R ) ) ) ) ) ) , $with ( $_L , $sha3_32 ( $add ( $_L , $uclamplt ( 2 , 3 ) ) ) , $with ( $_R , $add ( $mul ( 32 , $uclamplt ( 2 , 3 ) ) , $_R ) , $%seq ( $sstore ( $add ( $_L , 0 ) , $mload ( $add ( 0 , $_R ) ) ) ) ) ) ) ) ) , $sstore ( $add ( $_L , 1 ) , 5 ) ) ) , $stop ) ) ) , 0 ) ) )
